modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/settings/channels/pages/__tests__/ChannelList.jsx

```diff
@@ -3,12 +3,14 @@ import * as data from 'test/data';
 import ChannelList from '../ChannelList';
 import mockStore, {mockStoreData} from 'test/mock-store';
 import React from 'react';
-import {cleanup, render} from '@testing-library/react';
+import {cleanup, render, screen} from '@testing-library/react';
```

Comment:
Choose to use `screen` in all test or destructuring functions from `render`

Reply:
done